app/views/contest.scala.html

```diff
@@ -116,16 +116,16 @@ <h3>{{ events[e].name }}</h3>
                 <table class="table table-bordered result tribox-contest">
                     <thead>
                         <tr>
-                            <th>順位</th>
-                            <th>獲得SP</th>
+                            <th><i class="fa fa-hashtag"></i></th>
+                            <th>SP</th>
                             <th>名前</th>
                             <th class="mb-hide">所属</th>
-                            <th ng-if="e != 'e333fm' && (events[e].method == 'average' || events[e].method == 'mean')">平均記録</th>
-                            <th ng-if="e != 'e333fm' && events[e].method == 'best'">最高記録</th>
+                            <th ng-if="e != 'e333fm' && (events[e].method == 'average' || events[e].method == 'mean')">記録</th>
+                            <th ng-if="e != 'e333fm' && events[e].method == 'best'">記録</th>
                             <th ng-if="e == 'e333fm'">記録</th>
-                            <th class="mb-hide" ng-if="e != 'e333fm'">個々の記録</th>
-                            <th class="mb-hide" ng-if="e == 'e333fm'">解答・解説コメント</th>
-                            <th>使用キューブ</th>
+                            <th class="mb-hide" ng-if="e != 'e333fm'"></th>
+                            <th class="mb-hide" ng-if="e == 'e333fm'"></th>
```

Comment:
126-127行目も
```suggestion
                            <th class="mb-hide"></th>
```
でまとめちゃいたいです。